packages/rest-typings/src/v1/e2e.ts

```diff
@@ -87,6 +87,35 @@ const E2eSetRoomKeyIdSchema = {
 
 export const isE2eSetRoomKeyIdProps = ajv.compile<E2eSetRoomKeyIdProps>(E2eSetRoomKeyIdSchema);
 
+type E2EProvideUsersGroupKeyProps = {
+	usersSuggestedGroupKeys: Record<IRoom['_id'], { _id: IUser['_id']; key: string }[]>;
+};
+
+const E2EProvideUsersGroupKeySchema = {
+	type: 'object',
+	properties: {
+		usersSuggestedGroupKeys: {
+			type: 'object',
+			additionalProperties: {
```

Comment:
shouldn't this be `properties`?

Reply:
we don't have fixed properties, properties are roomIds which can be any random strings, let me check this.

Reply:
I always thought properties was for fixed props.

Reply:
It should be in properties for sure

Reply:
I was wrong, it's right to use additionalProperties in this case